migtests/scripts/yugabytedb/env.sh

```diff
@@ -1,7 +1,7 @@
 export TARGET_DB_HOST=${TARGET_DB_HOST:-"127.0.0.1"}
 export TARGET_DB_PORT=${TARGET_DB_PORT:-5433}
-export TARGET_DB_USER=${TARGET_DB_USER:-"yugabyte"}
-export TARGET_DB_PASSWORD=${TARGET_DB_PASSWORD:-''}
+export TARGET_DB_USER=${TARGET_DB_USER:-"ybvoyager"}
```

Comment:
Just the way we did with the source databases, define two sets of env vars:
- `TARGET_DB_ADMIN_USER` and `TARGET_DB_ADMIN_PASSWORD`
- `TARGET_DB_USER` and `TARGET_DB_USER`

The first pair is used for admin activities whereas second pair is used for actual migration.

Reply:
Done